.github/workflows/zephyr.yml

```diff
@@ -7,6 +7,13 @@ name: Zephyr
 # yamllint disable-line rule:truthy
 on: [push, pull_request, workflow_dispatch, workflow_call]
 
+# Specifies group name that stops previous wokrflows if the name matches
+concurrency:
+  # eg. "Zephyr-pull_request-my_fork_branch_to_merge"
+  # eg. "Zephyr-push-refs/heads/my_branch_merging"
```

Comment:
typoe: wokrflows

The examples are nice, help to review!

Reply:
Fixed 👍